### PR TITLE
chore(dui3): adds test bindings + a poc simple config binding

### PR DESCRIPTION
**Pull request by didimitrie**
Adds test bindings back + poc config binding
Tested in:
- Rhino 7
- Autocad

Not tested in (probably not needed, if any breakage it should be easily fly-by fixed later):
- ArcGIS
- Revit

tl;dr: i wanted to toggle dark mode, and had a heart attack when i noticed the sanity checks in the frontend were not all passing; realised the test binding was not in, so added that too (let's keep it there while we have those tests in the frontend/ui)

### CODE REVIEW

**Comment by didimitrie** [comment]
Pair PR in server: https://github.com/specklesystems/speckle-server/pull/2209 > Note, without this merged the config changes will not work and the UI might go full red in the console.

File: DUI3-DX/Connectors/ArcGIS/Speckle.Connectors.ArcGIS3/DependencyInjection/AutofacArcGISModule.cs [hunk 1]
```diff
@@ -34,6 +34,8 @@ protected override void Load(ContainerBuilder builder)
     builder.RegisterType<ArcGISDocumentStore>().SingleInstance();
 
     // Register bindings
+    builder.RegisterType<TestBinding>().As<IBinding>().SingleInstance();
+    builder.RegisterType<ConfigBinding>().As<IBinding>().SingleInstance().WithParameter("connectorName", "ArcGIS"); // POC: Easier like this for now, should be cleaned up later
```
Comment (BovineOx) [inline DUI3-DX/Connectors/ArcGIS/Speckle.Connectors.ArcGIS3/DependencyInjection/AutofacArcGISModule.cs:38]: I don't have too strong feeling about this, but arguable there may be a ArcGISContext that already has this string or could have it... certainly can live for a bit with the POC comment

File: DUI3-DX/DUI3/Speckle.Connectors.DUI/Bindings/ConfigBinding.cs [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+  public string Name => "configBinding";
+  public IBridge Parent { get; }
+  private SQLiteTransport ConfigStorage { get; }
+  private readonly string _connectorName;
+  private readonly JsonSerializerSettings _serializerOptions;
+
+  public ConfigBinding(IBridge bridge, JsonSerializerSettings serializerOptions, string connectorName)
+  {
+    Parent = bridge;
+    ConfigStorage = new SQLiteTransport(scope: "DUI3Config");
```
Comment (BovineOx) [inline DUI3-DX/DUI3/Speckle.Connectors.DUI/Bindings/ConfigBinding.cs:27]: Can this be injected yet? If not let's add a POC comment

**Review by BovineOx — CHANGES_REQUESTED** [review]
Mostly just adding the POC comment tbh

**Review by BovineOx — APPROVED** [review]

File: DUI3-DX/DUI3/Speckle.Connectors.DUI/Bindings/ConfigBinding.cs [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+    ConfigStorage.UpdateObject(_connectorName, str);
+  }
+}
+
+/// <summary>
+/// POC: A simple POCO for keeping track of settings. I see this as extensible in the future by each host application if and when we will need global per-app connector settings.
+/// </summary>
+public class ConnectorConfig
+{
+  public bool DarkTheme { get; set; } = true;
```
Comment (AlanRynne) [inline DUI3-DX/DUI3/Speckle.Connectors.DUI/Bindings/ConfigBinding.cs:75]: Unacceptable default 🤣

File: DUI3-DX/DUI3/Speckle.Connectors.DUI/Bindings/ConfigBinding.cs [hunk 4]
```diff
@@ -0,0 +1,76 @@
...
+
+namespace Speckle.Connectors.DUI.Bindings;
+
+/// <summary>
+/// POC: Simple config binding, as it was driving Dim nuts he couldn't swap to a dark theme.
+/// How does it store configs? In a sqlite db called 'DUI3Config', we create a row for each host application:
+/// [ hash,     contents         ]
+/// ['Rhino',   serialised config]
+/// ['Revit',   serialised config]
+/// </summary>
```
Comment (AlanRynne) [inline DUI3-DX/DUI3/Speckle.Connectors.DUI/Bindings/ConfigBinding.cs:15]: I'm not sure this is storing as it should, mine always starts in dark mode even after switching it.

**Review by AlanRynne — APPROVED** [review]
Looks good, but can we verify the settings are working right?

Should I expect "light mode" to be preserved in Rhino across restarts?

**Pull request merged**